build_tools/compiler/django_processor.rb

```diff
@@ -23,6 +23,7 @@ def self.statement_tag_for(key, default_value)
       footer_top:           block_for(:footer_top),
       homepage_url:         statement_tag_for(:homepage_url, 'https://www.gov.uk'),
       global_header_text:   statement_tag_for(:global_header_text, 'GOV.UK'),
+      fonts_stylesheets:    statement_tag_for(:fonts_stylesheets, '<!--[if IE 8]><link href="{% static \"fonts-ie8.css\" %}" media="all" rel="stylesheet" /><![endif]--><!--[if gte IE 9]><!--><link href="{% static \"fonts.css\" %}" media="all" rel="stylesheet" /><!--<![endif]-->'),
```

Comment:
There's a lot of repetition of the fonts stylesheet markup. Can you put that into a helper and have all the processors reference the same string.

This will also make bumping the font version easier.

Reply:
Good point, will do that